src/axom/core/examples/core_acceleration.cpp

```diff
@@ -0,0 +1,164 @@
+// Copyright (c) 2017-2021, Lawrence Livermore National Security, LLC and
+// other Axom Project Developers. See the top-level LICENSE file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+/*! \file core_containers.cpp
+ *  \brief This example code is a demonstration of the Axom Core containers.
+ */
+
+/* This example code contains snippets used in the Core Sphinx documentation.
+  * They begin and end with comments such as
+  *
+  * timer_start
+  * timer_end
+  *
+  * each prepended with an underscore.
+  */
+
+// Axom includes
+#include "axom/core/StackArray.hpp"
+#include "axom/core/execution/execution_space.hpp"
+#include "axom/core/execution/for_all.hpp"
+#include "axom/core/execution/synchronize.hpp"
+#include "axom/core/memory_management.hpp"
+
+#ifdef WIN32
+  #include "windows.h"
+void sleep(int numSeconds)
+{
+  int numMilliSecs = numSeconds * 1000;
+  Sleep(numMilliSecs);
+}
+#else
+  #include <unistd.h>  // for sleep()
+#endif
```

Comment:
Is this used?

If not, please remove it.

Reply:
removed.